grpc/src/main/java/com/linecorp/armeria/internal/client/grpc/GrpcClientCall.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2023 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.internal.client.grpc;
+
+import static java.util.Objects.requireNonNull;
+
+import com.linecorp.armeria.common.RequestContext;
+
+import io.grpc.CallOptions;
+import io.grpc.MethodDescriptor;
+import io.netty.util.AttributeKey;
+
+public final class GrpcClientCall {
```

Comment:
nit: Would it help to prepend `Internal` to the file name to make it easier to find in the IDE?
```suggestion
public final class InternalGrpcClientCall {
```